docs/tutorials/python/tutorial_scripts/json_schema.py

```diff
@@ -101,69 +100,47 @@
 # 4. Bind the JSON schema to the folder
 schema_uri = ORG_NAME + "-" + SCHEMA_NAME + "-" + VERSION
 bound_schema = test_folder.bind_schema(
-    json_schema_uri=schema_uri, synapse_client=syn, enable_derived_annotations=True
+    json_schema_uri=schema_uri, enable_derived_annotations=True
 )
 json_schema_version_info = bound_schema.json_schema_version_info
-client.logger.info("JSON schema was bound successfully. Please see details below:")
+syn.logger.info("JSON schema was bound successfully. Please see details below:")
 pprint(vars(json_schema_version_info))
 
 # 5. Retrieve the Bound Schema
 schema = test_folder.get_schema()
-client.logger.info("JSON Schema was retrieved successfully. Please see details below:")
+syn.logger.info("JSON Schema was retrieved successfully. Please see details below:")
 pprint(vars(schema))
 
 # 6. Add Invalid Annotations to the Folder and Store
 test_folder.annotations = {
     "patient_id": "1234",
     "cognitive_score": "invalid str",
 }
-test_folder.store(synapse_client=syn)
+test_folder.store()
 
 time.sleep(2)
 
 validation_results = test_folder.validate_schema()
-client.logger.info("Validation was completed. Please see details below:")
+syn.logger.info("Validation was completed. Please see details below:")
 pprint(vars(validation_results))
 
 # 7. Create a File with Invalid Annotations and Upload It
 # Then, view validation statistics and invalid validation results
-if not os.path.exists(os.path.expanduser("~/temp")):
-    os.makedirs(os.path.expanduser("~/temp/testJSONSchemaFiles"), exist_ok=True)
-
-name_of_file = "test_file.txt"
-path_to_file = os.path.join(
-    os.path.expanduser("~/temp/testJSONSchemaFiles"), name_of_file
-)
-
-
-def create_random_file(
-    path: str,
-) -> None:
-    """Create a random file with random data.
-
-    :param path: The path to create the file at.
-    """
-    with open(path, "wb") as f:
-        f.write(os.urandom(1))
-
-
-create_random_file(path_to_file)
+path_to_file = make_bogus_data_file(n=5)
```

Comment:
out-of-scope, but do we want to rename the parameter `n` in this helper function to something more descriptive?

Reply:
Users shouldn't be using this in any regular code. The argument is explained well enough that I don't see us needing to make a change: https://python-docs.synapse.org/en/stable/reference/core/#synapseclient.core.utils.make_bogus_data_file

Ideally users would be replacing this line to point to a file they want to upload.

This disclaimer is also on that doc page:
`This section is for super users / developers only. These functions are subject to change as they are internal development functions. Use at your own risk.`